packages/applicant/src/pages/applications/index.page.tsx

```diff
@@ -153,6 +114,79 @@ const ExistingApplications = ({ applicationData }: ApplicationsPage) => {
   );
 };
 
+const ApplicationRow = (application) => {
+  const applicationName = application.applicationName;
+  const submissionId = application.grantSubmissionId;
+  const applicationStatusTag =
+    APPLICATION_STATUS_TAGS[application.submissionStatus];
+  const isSubmitted = application.submissionStatus === 'SUBMITTED';
+  const applicationLinkText = isSubmitted ? 'View' : 'Edit';
+  const applicationLink = isSubmitted
+    ? '/apply/applicant' + routes.submissions.summary(submissionId)
+    : '/apply/applicant' + routes.submissions.sections(submissionId);
+  return (
+    <tr key={submissionId} className="govuk-table__row">
+      <th scope="row" className="govuk-table__cell">
+        <p
+          className="govuk-!-margin-0 govuk-!-font-weight-bold"
+          data-cy={`cy-application-link-${applicationName}`}
+        >
+          {applicationName}
+        </p>
+      </th>
+      <td
+        scope="row"
+        className="govuk-table__cell"
+        aria-describedby={`status-tag-${submissionId}`}
+      >
+        <strong
+          className={`govuk-tag ${applicationStatusTag.colourClass}`}
+          data-cy={`cy-status-tag-${applicationName}-${applicationStatusTag.displayName}`}
+          id={`status-tag-${submissionId}`}
+        >
+          {applicationStatusTag.displayName}
+        </strong>
+      </td>
+      <td
+        scope="row"
+        className="govuk-table__cell"
+        aria-describedby={`submitted-date-${submissionId}`}
+      >
+        <p
+          className="govuk-!-margin-0 govuk-!-font-weight-normal"
+          data-cy={`cy-application-submitted-date-${applicationName}`}
+          id={`submitted-date-${submissionId}`}
+        >
+          {application.submittedDate
+            ? moment(application.submittedDate).format('D MMMM YYYY')
```

Comment:
I know this is not the PR to fix this issue but we really need to get rid of `moment`, iirc it adds like 360kb to uncompressed bundle size